src/Components/ServiceSelectionScene/ServiceSelectionScene.tsx

```diff
@@ -29,23 +37,35 @@ import { getFavoriteServicesFromStorage } from 'services/store';
 import {
   getDataSourceVariable,
   getLabelsVariable,
+  getServiceNameVariable,
   getServiceSelectionStringVariable,
   LEVEL_VARIABLE_VALUE,
   SERVICE_NAME,
+  SERVICE_NAME_EXPR,
+  SERVICE_NAME_VAR,
   VAR_SERVICE,
   VAR_SERVICE_EXPR,
 } from 'services/variables';
 import { selectService, SelectServiceButton } from './SelectServiceButton';
 import { buildDataQuery, buildResourceQuery } from 'services/query';
 import { reportAppInteraction, USER_EVENTS_ACTIONS, USER_EVENTS_PAGES } from 'services/analytics';
-import { getQueryRunner, setLevelColorOverrides } from 'services/panel';
+import { getQueryRunner, getSceneQueryRunner, setLevelColorOverrides } from 'services/panel';
 import { ConfigureVolumeError } from './ConfigureVolumeError';
 import { NoVolumeError } from './NoVolumeError';
 import { getLabelsFromSeries, toggleLevelVisibility } from 'services/levels';
 import { ServiceFieldSelector } from '../ServiceScene/Breakdowns/FieldSelector';
 import { CustomConstantVariable } from '../../services/CustomConstantVariable';
 import { areArraysEqual } from '../../services/comparison';
-
+import { config, getAppEvents } from '@grafana/runtime';
+import { VariableHide } from '@grafana/schema';
+import { PLUGIN_ID } from '../../services/routing';
+
+// @ts-expect-error
+const aggregatedMetricsEnabled: boolean | undefined = config.featureToggles.exploreLogsAggregatedMetrics;
+// Don't export AGGREGATED_SERVICE_NAME, we want to rename things so the rest of the application is agnostic to how we got the services
+const AGGREGATED_SERVICE_NAME = '__aggregated_metric__';
+const AGGREGATED_METRIC_START_DATE = dateTime('2024-08-30', 'YYYY-MM-DD');
```

Comment:
eventually we may want this to be configurable (ie. OSS users), but I'm totally fine with this for now

Reply:
Agreed, @svennergr is there a way we can configure this in deployment tools with an env variable or something so OSS users can set their own date?